smithy-aws-protocol-tests/model/rpcV2/cbor-lists.smithy

```diff
@@ -0,0 +1,311 @@
+        }
+    }
+])
+
+apply RpcV2CborLists @httpResponseTests([
+    {
+        id: "RpcV2CborLists",
+        documentation: "Serializes RpcV2 Cbor lists",
+        protocol: rpcv2Cbor,
+        code: 200,
+        body: "v2tib29sZWFuTGlzdJ/19P9oZW51bUxpc3SfY0Zvb2Ew/2tpbnRFbnVtTGlzdJ8BAv9raW50ZWdlckxpc3SfAQL/cG5lc3RlZFN0cmluZ0xpc3Sfn2Nmb29jYmFy/59jYmF6Y3F1eP//anN0cmluZ0xpc3SfY2Zvb2NiYXL/aXN0cmluZ1NldJ9jZm9vY2Jhcv9tc3RydWN0dXJlTGlzdJ+/YWFhMWFiYTL/v2FhYTNhYmE0//9tdGltZXN0YW1wTGlzdJ/B+0HU1/vzgAAAwftB1Nf784AAAP//",
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Content-Type": "application/cbor"
+        },
+        params: {
+            "stringList": [
+                "foo",
+                "bar"
+            ],
+            "stringSet": [
+                "foo",
+                "bar"
+            ],
+            "integerList": [
+                1,
+                2
+            ],
+            "booleanList": [
+                true,
+                false
+            ],
+            "timestampList": [
+                1398796238,
+                1398796238
+            ],
+            "enumList": [
+                "Foo",
+                "0"
+            ],
+            "intEnumList": [
+                1,
+                2
+            ],
+            "nestedStringList": [
+                [
+                    "foo",
+                    "bar"
+                ],
+                [
+                    "baz",
+                    "qux"
+                ]
+            ],
+            "structureList": [
+                {
+                    "a": "1",
+                    "b": "2"
+                },
+                {
+                    "a": "3",
+                    "b": "4"
+                }
+            ]
+        }
+    },
+    {
+        id: "RpcV2CborListsEmpty",
+        documentation: "Serializes empty RpcV2 Cbor lists",
+        protocol: rpcv2Cbor,
+        code: 200,
+        body: "v2pzdHJpbmdMaXN0n///",
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Content-Type": "application/cbor"
+        },
+        params: {
+            stringList: []
+        }
+    },
+    {
+        id: "RpcV2CborListsSerializeNull",
+        documentation: "Serializes null values in sparse lists",
+        protocol: rpcv2Cbor,
+        code: 200,
+        body: "v3BzcGFyc2VTdHJpbmdMaXN0n/ZiaGn//w==",
+        bodyMediaType: "application/cbor",
+        headers: {
+            "smithy-protocol": "rpc-v2-cbor",
+            "Content-Type": "application/cbor"
+        },
+        params: {
+            sparseStringList: [null, "hi"]
+        }
+    }
+])
+
+structure RpcV2CborListInputOutput {
+    stringList: StringList,
+
+    sparseStringList: SparseStringList,
```

Comment:
Can all `@sparse` collections be split to their own operations, similar to #2206 ?

Reply:
They have been moved into separate operation.